templates/go/greeter.go

```diff
@@ -1,14 +1,33 @@
 package main
 
 import (
-	"fmt"
-
 	restate "github.com/restatedev/sdk-go"
+	"time"
 )
 
 // Greeter is a struct which represents a Restate service; reflection will turn exported methods into service handlers
 type Greeter struct{}
 
-func (Greeter) Greet(ctx restate.Context, greeting string) (string, error) {
-	return fmt.Sprintf("%s!", greeting), nil
+func (Greeter) Greet(ctx restate.Context, name string) (string, error) {
+    //  Durably execute a set of steps; resilient against failures
+	greetingId := restate.Rand(ctx).UUID().String()
+
+	if _, err := restate.Run(ctx, func(ctx restate.RunContext) (restate.Void, error) {
```

Comment:
I think this should work fine now without `restate.Void`, just

```suggestion
	if _, err := restate.Run(ctx, func(ctx restate.RunContext) error {
```

Reply:
that doesn't work for me with v0.12.0